src/analysis/normed_space/operator_norm.lean

```diff
@@ -642,6 +639,48 @@ begin
       ... ≤ ∥smul_right c f∥ * ∥x∥ : le_op_norm _ _ } },
 end
 
+/-- Given `c : c : E →L[𝕜] 𝕜`, `c.smul_rightₗ` is the linear map from `F` to `E →L[𝕜] F`
+sending `f` to `λ e, c e • f`. See also `continuous_linear_map.smul_rightL`. -/
+def smul_rightₗ (c : E →L[𝕜] 𝕜) : F →ₗ[𝕜] (E →L[𝕜] F) :=
```

Comment:
Could this (and some of the lemmas that follow) be moved to `topology.algebra.module` (i.e. to the setting of topological rather than normed vector spaces)?

Reply:
I moved that definition but the next ones require a topology on `E →L[𝕜] F` that I don't think we don't have if `E` and `F` are not normed.

Reply:
I see, thanks for taking the idea this far!